__tests__/test-data.test.js

```diff
@@ -50,3 +48,35 @@ describe("api/topics", () => {
     );
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("returns article api ", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("200,check if it returns an array ", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const arrayOfArticles = response.body.data;
+        expect(Array.isArray(arrayOfArticles)).toBe(true);
+      });
+  });
+  test("200,returns the correct obj ", () => {
+    return request(app)
```

Comment:
This test is only checking the correct id is returned, you could check the entire object as that is what your test description says.